ci/before_script.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+  
```

Comment:
Please add `set -xeo pipefail` after the shebang.

Reply:
done